packages/access-token/tsconfig.json

```diff
@@ -1,6 +1,9 @@
 // This file can contain .js-specific Typescript compiler config.
 {
   "extends": "../../tsconfig.json",
+  "compilerOptions": {
+    "allowSyntheticDefaultImports": true
```

Comment:
Why is this needed ?

Reply:
It's needed to silence my IDE from complaining about `import path from 'path'` and everything like it.

Reply:
`path` is surprising, but `proper-lockfile` I could understand. See comment on that import, and revert this if it does the trick.

Reply:
Doesn't do the trick, so I'm keeping it.

Reply:
Oh I never realized that other packages have this option too! My bad